### DISCUSSION TITLE
two cursors appearing instead of one

### DISCUSSION THREAD

**Discussion by raybeanosu** [ppy/osu#16845] [Q&A] [answered]
**logs here**: [message (2).txt](https://github.com/ppy/osu/files/8040224/message.2.txt)
posting this for a friend who uses a tablet, and is seeing two cursors instead of one. this is what she said,

> so i use my stylus and suddenly i get two spammy cursors, so when i move it around it either clicks stuff or obviously doesnt get the circles

the double cursors go away when switching from tablet to mouse, but returns back again on switching back, unplugging mouse doesn't make a difference either.

a reddit help thread suggested reinstalling osu, but that did not work as well.
![20220210_160520](https://user-images.githubusercontent.com/91675300/153390944-40a2aa86-09fa-4e1a-b77b-69fbfff25617.jpg)

![20220210_160531](https://user-images.githubusercontent.com/91675300/153390954-d5379a8f-588a-4e58-9e7c-aae1cd8cc911.jpg)

**Comment by integerrr** [comment, marked as answer]
This repository is for osu!lazer development, please post this issue to the other repo which is tracking stable issues: https://github.com/ppy/osu-stable-issues

**Reply by raybeanosu** [reply]
sure, i don't really use github though, how do i repost this? do i just give the # for this issue, or rewrite this there?

**Reply by integerrr** [reply]
Just copy and pasting this post over there seems reasonable to me, do check if there's similar issues posted already to avoid posting duplicates

**Reply by raybeanosu** [reply]
aight, will do!
